apps/func-response-processor/src/index.ts

```
@@ -358,6 +359,71 @@ const serviceBusTrigger = async function (
+
+        if (participantData) {
+          const pointsPercentage = gradeQuestionSelection({
+            numberOfInputs: parseInt(instanceInfo.numberOfInputs),
+            response: response.selection,
+            correctAnswers: parsedSolutions,
+          })
+
+          pointsAwarded = computeAwardedPoints({
+            firstResponseReceivedAt,
+            responseTimestamp,
+            maxBonus: isNaN(parseInt(instanceInfo.maxBonusPoints, 10))
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)